README.md

```diff
@@ -14,12 +14,22 @@ Available variables are listed below, along with default values (see `defaults/m
 
     fluentd_package_state: present
 
+Extra groups to add log access, secret:
+    fluentd_groups:
```

Comment:
Your naming `extra groups` speaks by itself:
```suggestion
    fluentd_extra_groups:
```

Reply:
I've hesitated between both in fact